### ISSUE TITLE
[Windows] Unable to download stickers - Python TypeError

### ISSUE THREAD

**Issue by unkn0wnAPI** [laggykiller/sticker-convert#44]
## **Bug Description**

Unable to download stickers, due to Python `TypeError: join() argument must be str, bytes, or os.PathLike object, not 'File'`. <br>Disclaimer: **Tested only with telegram stickers**

---

### **Steps To Reproduce**

1. Download sticker-convert-windows.zip (v1.19.2);
2. Add necessary credentials;
3. Try to download a sticker pack.

---

### **Expected behaviour**

The sticker pack should be downloaded, without any problems.

---

### **Console Output**

```
Launching GUI...
Skip moving old files in input directory as input source is empty
Skip moving old files in output directory as output source is empty
Downloading...
Exception in thread Thread-2 (start_process):
Traceback (most recent call last):
  File "threading.py", line 1038, in _bootstrap_inner
  File "threading.py", line 975, in run
  File "gui.py", line 348, in start_process
  File "flow.py", line 61, in start
  File "flow.py", line 270, in download
  File "downloaders\download_telegram.py", line 70, in download_stickers_telegram
  File "anyio\_core\_eventloop.py", line 68, in run
  File "anyio\_backends\_asyncio.py", line 204, in run
  File "asyncio\runners.py", line 190, in run
  File "asyncio\runners.py", line 118, in run
  File "asyncio\base_events.py", line 653, in run_until_complete
  File "anyio\_backends\_asyncio.py", line 199, in wrapper
  File "downloaders\download_telegram.py", line 53, in download_stickers_telegram_async
  File "<frozen ntpath>", line 147, in join
  File "<frozen genericpath>", line 152, in _check_arg_types
TypeError: join() argument must be str, bytes, or os.PathLike object, not 'File'
```

---

### **My environment**

* OS: Windows 10 22H2 Pro 
* Sticker convert version: v1.19.2

**Comment by laggykiller** [comment]
Thank you for your detailed and easy-to-read bug report. Fixed in v1.20.0. Please check.

**Comment by unkn0wnAPI** [comment]
Thank you for the swift fix, version v1.20.0 fixed the issue.
